execution/function/functions.go

```diff
@@ -66,6 +66,21 @@ var Funcs = map[string]FunctionCall{
 	"asinh": simpleFunc(math.Asinh),
 	"acosh": simpleFunc(math.Acosh),
 	"atanh": simpleFunc(math.Atanh),
+	"absent": func(f FunctionArgs) promql.Sample{
```

Comment:
I guess we need to fix the series labels. Please refer to upstream Prometheus and see how it implements absent https://github.com/prometheus/prometheus/blob/main/promql/functions.go#L591

Reply:
should i try to implement functions in promql-engine in similar way as promql function in prometheus